### PR TITLE
+Chromebook and ARM terms

### PR DESCRIPTION
**Pull request by NameLessGO**
Helpful for users searching for Chromebook and ARM

### CODE REVIEW

**Comment by NameLessGO** [comment]
What you think about the Aarch64 commit

it is not future-proof as the Aarch64 can be replaced in the future

File: src/platform/linux/installing.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 The packaged version requires a recent 64 bit Intel/AMD Linux with glibc, and common
 libraries like libwayland-client and systemd. If you are on a different
-architecture, or a barebones Linux distro, you will not be able to use the
+architecture (e.g Arm/Aarch64), or a barebones Linux distro, you will not be able to use the
```
Comment (dae) [inline src/platform/linux/installing.md:9]:
```suggestion
architecture (e.g ARM/AArch64), or a barebones Linux distro, you will not be able to use the
```

File: src/platform/linux/installing.md [hunk 2]
```diff
@@ -6,11 +6,11 @@
...
 packaged version, but you may be able to use the [Python wheels](https://betas.ankiweb.net/#via-pypipip)
 instead.
 
-On recent Ubuntu installations, please use the following before
+Debian and derivatives, Namely Ubuntu and Chromebook, please use the following before
```
Comment (dae) [inline src/platform/linux/installing.md:13]:
```suggestion
Debian and derivatives, such as Ubuntu and Chromebooks with Linux enabled, please use the following before
```

It appears that Linux is not enabled out of the box.

https://support.google.com/chromebook/answer/9145439?hl=en

**Comment by NameLessGO** [comment]
Added google support enable Linux link to `Chromebook with linux`

**Comment by dae** [comment]
Thanks!

**Pull request merged**